schema/event_mapping_entity.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$udmi_version": "1.4.0",
+  "title": "Mapping Event Entity",
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "code": {
```

Comment:
"code" isn't what I'd expect here... "name"?  "id"?    I wouldn't call something like "fcu-8_1" a code...

Reply:
"code" is from the DBO definition "Code: The human readable identifier for the entity. This should be unique in document scope." https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md

@grafnu does this work or would you still prefer a change to something like "name", "id", or even "humanReadableId" ?

Reply:
code is fine -- just wasn't sure where it came from!